test_global_controller/genesis/run.csh

```diff
@@ -0,0 +1,12 @@
+Genesis2.pl -parse -generate -top top -input 		top.svp \
```

Comment:
can we get rid of this file? do we need it?

Reply:
No, that file is the top of the testbench, so it's needed. I could rename it to test_top.svp if that's less confusing.

Reply:
Sorry I mean to ask if `run.csh` is needed at all. I don't see it being used.

Reply:
oops, yeah I can remove that.